app/controllers/fs/files_controller.rb

```diff
@@ -8,77 +8,77 @@ class Fs::FilesController < ApplicationController
   rescue_from StandardError, with: :rescue_action
 
   private
-    def set_item
-      id = params[:id_path].present? ? params[:id_path].gsub(/\//, "") : params[:id]
-      path = params[:filename]
-      path << ".#{params[:format]}" if params[:format].present?
+  def set_item
+    id = params[:id_path].present? ? params[:id_path].gsub(/\//, "") : params[:id]
+    path = params[:filename]
+    path << ".#{params[:format]}" if params[:format].present?
 
-      @item = SS::File.find_by id: id, filename: path
-      raise "404" if @item.thumb?
-    end
+    @item = SS::File.find_by id: id, filename: path
+    raise "404" if @item.thumb?
+  end
 
-    def deny
-      return if @item.public?
-      return if SS.config.env.remote_preview
+  def deny
+    return if @item.public?
+    return if SS.config.env.remote_preview
 
-      user   = get_user_by_session
-      member = get_member_by_session
-      item   = @item.becomes_with_model
-      raise "404" unless item.previewable?(user: user, member: member)
+    user   = get_user_by_session
+    member = get_member_by_session
+    item   = @item.becomes_with_model
+    raise "404" unless item.previewable?(user: user, member: member)
 
-      set_last_logged_in
-    end
+    set_last_logged_in
+  end
 
-    def set_last_modified
-      response.headers["Last-Modified"] = CGI::rfc1123_date(@item.updated.in_time_zone)
-    end
+  def set_last_modified
+    response.headers["Last-Modified"] = CGI::rfc1123_date(@item.updated.in_time_zone)
+  end
 
-    def rescue_action(e = nil)
-      if e.to_s =~ /^\d+$/
-        status = e.to_s.to_i
-        file = error_html_file(status)
-        return ss_send_file(file, status: status, type: Fs.content_type(file), disposition: :inline)
-      end
-      raise e
+  def rescue_action(e = nil)
+    if e.to_s =~ /^\d+$/
+      status = e.to_s.to_i
+      file = error_html_file(status)
+      return ss_send_file(file, status: status, type: Fs.content_type(file), disposition: :inline)
     end
+    raise e
+  end
 
-    def error_html_file(status)
-      file = "#{Rails.public_path}/#{status}.html"
-      Fs.exists?(file) ? file : "#{Rails.public_path}/500.html"
-    end
+  def error_html_file(status)
+    file = "#{Rails.public_path}/#{status}.html"
+    Fs.exists?(file) ? file : "#{Rails.public_path}/500.html"
+  end
 
   public
-    def index
-      set_last_modified
+  def index
+    set_last_modified
 
-      if Fs.mode == :file && Fs.file?(@item.path)
-        send_file @item.path, type: @item.content_type, filename: @item.filename,
-          disposition: :inline, x_sendfile: true
-      else
-        send_data @item.read, type: @item.content_type, filename: @item.filename,
-          disposition: :inline
-      end
+    if Fs.mode == :file && Fs.file?(@item.path)
+      send_file @item.path, type: @item.content_type, filename: @item.filename,
```

Comment:
__[brakeman]__
* Model attribute used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

<!-- Comment by [SideCI](https://sideci.com) -->